packages/vuetify/src/components/VProgressLinear/_mixins.sass

```diff
@@ -0,0 +1,47 @@
+@keyframes indeterminate
```

Comment:
Could these not go in variables file as well?

Reply:
Undecided about this.

Reply:
These aren't mixins